portfolio/src/main/webapp/hobbies/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Quentin Ross' Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <h1>Outside of typing code</h1>
+      <p>
+          My daily life consists of a lot of different hobbies and interests. 
```

Comment:
Feel free to use br tag to separate lines. 
https://www.w3schools.com/tags/tag_br.asp